monGARS/core/llm_integration.py

```
@@ -14,9 +14,17 @@
 from urllib.parse import urlparse
 
-import httpx
-from opentelemetry import metrics
-from transformers import AutoModelForCausalLM, AutoTokenizer, BitsAndBytesConfig
+import httpx
+from opentelemetry import metrics
+from transformers import AutoModelForCausalLM, AutoTokenizer, BitsAndBytesConfig
+
+try:  # pragma: no cover - optional dependency used for local dispatch
+    import ollama  # type: ignore
+except Exception as exc:  # pragma: no cover - optional dependency
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)